backup.md

```diff
@@ -0,0 +1,157 @@
+toc: false
+---
+
+{{site.data.alerts.callout_danger}}The <code>BACKUP</code> feature is only available to <a href="https://www.cockroachlabs.com/pricing/">enterprise license</a> users. For non-enterprise backups, see <a href="sql-dump.html"><code>cockroach dump</code></a>.{{site.data.alerts.end}}
+
+CockroachDB's `BACKUP` [statement](sql-statements.html) creates full or incremental backups of your cluster's schemas and data that are consistent as of a given timestamp. These backups can be stored on the platforms you're already using, including AWS S3, Google Cloud Storage, NFS, or HTTP storage.
+
+Because CockroachDB is designed with high fault tolerance, these backups are designed primarily for disaster recovery (i.e. if your cluster loses a majority of its nodes) through [`RESTORE`](restore.html). Isolated issues (such as small-scale node outages) do not require any intervention.
+
+<div id="toc"></div>
+
+## Functional Details
+
+### Backup Targets
+
+You can backup entire tables (which automatically includes their indexes) or [views](views.html) from a backup. Backing up a database simply backs up all of its tables and views.
+
+{{site.data.alerts.callout_info}}<code>BACKUP</code> <em>does not</em> support row-level backups.{{site.data.alerts.end}}
+
+### Object Relationships
+
+All objects with relationships must be backed up at the same time.
+
+Relationships include:
+
+- Tables with foreign key constraints and the tables they reference (although this relationship can instead be [removed during the restore](restore.html#skipmissingforeignkeys), letting you restore only one of the tables)
+- [Views](views.html) and the tables they depend on
+- [Interleaved tables](interleave-in-parent.html)
+
+### Users and Privileges
+
+To back up users and their passwords, you must specifically back up `system.users`. To restore it, you must use [this procedure](restore.html#restoring-users-from-systemusers-backup).
+
+You cannot back up privileges, though when you restore tables or views they inherit the privileges granted on the target database. Table-level privileges must be [granted to users](grant.html) after the restore is complete.
+
+### Backup Types
+
+CockroachDB offers two types of backups: full and incremental.
+
+#### Full Backups
+
+Full backups contain an unreplicated copy of your data and can always be used to restore your cluster.
+
+#### Incremental Backups
+Incremental backups contain only the data that has changed since a base set of backups you specify (which must include one full backup, and can include many incremental backups), so they're smaller and faster to produce. 
+
+You can only create incremental backups within the garbage collection period of the base backup's most recent timestamp. This is because incremental backups are created by comparing the cluster's current data with the data as it appeared at the most recent timestamp in the base backups––that timestamp data, though, is deleted by the garbage collection process.
```

Comment:
Not *quite* true:
Incremental backups don't actually compare to the backup data (which implies they need to read the whole backup) -- they only look up the time of that backup, then look at the change records in the database itself since that time, and it is the change records that are GC'ed.

somewhat pedantic nit, but wanted to avoid implication that it needed to re-read the whole backup to do comparisons.